src/FakeItEasy/Captured.of.TArgument.TCapture.cs

```diff
@@ -70,12 +70,12 @@ public class Captured<TArgument, TCapture>
         /// </exception>
         public TCapture GetLastValue()
         {
-            if (this.values.Any())
+            if (this.values.Count == 0)

```

Comment:
As you can see, I preferred to make a messier diff to preserve "positive comparisons". Can switch if you prefer. Just means you'll have to read the changes again.